# for air

float32 airtemperature
float32 airhumidity
